### ISSUE TITLE
Error on https://www.addictinggames.com/shooting/dragon-slayers

### ISSUE THREAD

**Issue by whobiggs** [ruffle-rs/ruffle#8036]
# Error Info
Error name: Error
Error message: panicked at 'index out of bounds: the len is 15 but the index is 15', /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/lyon_tessellation-1.0.0/src/fill.rs:1547:27
Error stack:
```
Error: panicked at 'index out of bounds: the len is 15 but the index is 15', /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/lyon_tessellation-1.0.0/src/fill.rs:1547:27
    at V.e.wbg.__wbg_new_df6e6ab7a65c4c4d (https://www.addictinggames.com/sites/all/themes/starkish/build/js/ruffle/core.ruffle.028d130f721799cfe9dd.js:1:6065)
    at std::sync::once::Once::call_once::{{closure}}::h96c2b4473ff6d536 (wasm://wasm/01688442:wasm-function[1104]:0x0)
    at std::sync::once::Once::call_inner::h1c414562e2619117 (wasm://wasm/01688442:wasm-function[1420]:0x0)
    at core::ops::function::FnOnce::call_once{{vtable.shim}}::h14edbfd8adf99b9a (wasm://wasm/01688442:wasm-function[4685]:0x0)
    at std::panicking::rust_panic_with_hook::h1c368a27f9b0afe1 (wasm://wasm/01688442:wasm-function[3116]:0x0)
    at core::panicking::panic_fmt::hcdb13a4b2416cf82 (wasm://wasm/01688442:wasm-function[4866]:0x0)
    at core::panicking::panic_bounds_check::he79659661cacde3e (wasm://wasm/01688442:wasm-function[4180]:0x0)
    at lyon_tessellation::fill::FillTessellator::process_events::hf251f5e99b1bb5f8 (wasm://wasm/01688442:wasm-function[354]:0x0)
    at lyon_tessellation::fill::FillTessellator::tessellate_impl::h32fa761dd0b10154 (wasm://wasm/01688442:wasm-function[546]:0x0)
    at ruffle_render_webgl::WebGlRenderBackend::register_shape_internal::h38b7a1aa623d20f4 (wasm://wasm/01688442:wasm-function[649]:0x0)
```

# Player Info
Allows script access: true
Player type: Object
SWF URL: /sites/default/files/flash_games/dragonslayers_0.swf
Param menu: false
Param allowscriptaccess: always
Param allownetworking: all
Param movie: /sites/default/files/flash_games/dragonslayers_0.swf
Attribute 0: undefined
Attribute 1: undefined
Attribute 2: undefined
Attribute 3: undefined
Attribute 4: undefined

# Page Info
Page URL: https://www.addictinggames.com/shooting/dragon-slayers
SWF URL: https://www.addictinggames.com/sites/default/files/flash_games/dragonslayers_0.swf

# Browser Info
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36 Edg/105.0.1343.27
Platform: Win32
Has touch support: false

# Ruffle Info
Version: 0.1.0
Name: nightly 2022-07-25
Channel: nightly
Built: 2022-07-25T00:17:23.429Z
Commit: afb9ecc4ab8a6f6b9b295ddcb029a4ca80230a9e
Is extension: false

**Comment by whobiggs** [comment]
I suspect it's to do with Flash but I'm not sure. Is there a way of running Flash games on Windows now?

**Comment by n0samu** [comment]
@whobiggs this is just a bug in your specific version of Microsoft Edge 105. Updating to the next version of Edge should fix it! See #7849 (different game, but same problem).
